plugins/chat/spec/models/chat_message_spec.rb

```diff
@@ -570,6 +570,68 @@
     end
   end
 
+  describe "#create_mentions" do
+    fab!(:message) { Fabricate(:chat_message) }
+    fab!(:user1) { Fabricate(:user) }
+    fab!(:user2) { Fabricate(:user) }
+
+    it "creates mentions for passed user ids" do
+      mentioned_user_ids = [user1.id, user2.id]
+      message.create_mentions(mentioned_user_ids)
+      message.reload
+
+      expect(message.chat_mentions.pluck(:user_id)).to contain_exactly(*mentioned_user_ids)
```

Comment:
If you use `match_array` (which is the same as `contain_exactly`) you don't have to splat (*) the `mentioned_user_ids`